Kiosk/Bid Fulfillment/LoadingViewController.swift

```diff
@@ -137,6 +137,10 @@ public class LoadingViewController: UIViewController {
         titleLabel.text = "Registration Complete"
         bidConfirmationImageView.image = UIImage(named: "BidHighestBidder")
         fulfillmentContainer()?.cancelButton.setTitle("DONE", forState: .Normal)
+        RACSignal.interval(1, onScheduler: RACScheduler.mainThreadScheduler()).take(1).subscribeCompleted { [weak self] () -> Void in
```

Comment:
I've seen this pattern a lot, maybe it should be turned into something with a name that actually represents what it means?